### PR TITLE
Issue #13100: Create tests from ParenPad xdoc examples

### PR DESCRIPTION
**Pull request by stoyanK7**
Part of #13100

https://checkstyle.sourceforge.io/config_whitespace.html#ParenPad

### CODE REVIEW

**Review by romani — APPROVED** [review]
Ok to merge

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ParenPad
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.parenpad;
+
+class Example1 {
```
Comment (nrmancuso) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java:9]: Was leaving out xdoc comments intentional?

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java (hunk 1 above)
Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java:9]: So you mean  missed `// xdoc section -- start` ?

Probably not intentional, but still can be done on the moment we will make them actionable.

Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java:10]: Yes. I completely missed it. Added back. Done.

Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java:10]: @stoyanK7 , do we have this in ideas for validation by checkstyle? Please add to that ticket if missed. I think all of them will have such comments.

Comment (nrmancuso) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java:10]: @romani added to https://github.com/checkstyle/checkstyle/issues/13244#issue-1761844973

**Review by nrmancuso — APPROVED** [review]

**Comment by nrmancuso** [comment]
Failure in semaphore is not related to changes here.

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java (hunk 1 above)
Comment (stoyanK7) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java:10]: We already have validation for that. It got merged a few hours ago.

https://github.com/checkstyle/checkstyle/blob/8057f64b06848ced41ff3d53438d184999e8c94a/config/checkstyle-examples-checks.xml#L47-L60

**Pull request merged**